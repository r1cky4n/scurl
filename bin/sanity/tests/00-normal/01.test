https://rsa8192.badssl.com/
