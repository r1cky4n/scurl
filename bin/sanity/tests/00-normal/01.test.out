<!DOCTYPE html>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" href="/icons/favicon-yellow.ico"/>
  <link rel="apple-touch-icon" href="/icons/icon-yellow.png"/>
  <title>rsa8192.badssl.com</title>
  <link rel="stylesheet" href="/style.css">
  <style>body { background: rgb(246, 207, 47); }</style>
</head>
<body>
<div id="content">
  <h1 style="font-size: 8vw;">
    rsa8192.badssl.com
  </h1>
</div>

<div id="footer" style="font-size: 2.75vw;">
  This site uses an 8192-bit RSA key for key exchange.<br>
  (8192-bit RSA keys were <a href="http://blog.shiz.me/post/67305143330/8192-bit-rsa-keys-in-os-x">not supported</a> in OSX between 2006 and <a href="https://twitter.com/ivanristic/status/662581577835114497">2015</a>.)
</div>

</body>
</html>
